tools/bumper/bumper.go

```diff
@@ -0,0 +1,139 @@
+		}
+
+		//TODO updatePolicyDummy is a placeholder. In future task will be replaced by proper parameter set in the configYaml
+		updatePolicyDummy := "static"
+		bumpNeeded, err := isBumpNeeded(currentReleaseVersion, latestReleaseVersion, updatePolicyDummy)
+		if err != nil {
+			exitWithError(errors.Wrapf(err, "Failed to discover if Bump need for %s", componentName))
+		}
+
+		if bumpNeeded {
+			Logger.Printf("Bumping %s from %s to %s", componentName, currentReleaseVersion, latestReleaseVersion)
+			//reset --hard git repo
+
+			//PR name
+			PRTitle := fmt.Sprintf("Bump %s to %s", componentName, latestReleaseVersion)
+			Logger.Printf("PR title %s", PRTitle)
+			//Create PR
+
+			//update component's entry in config yaml
+			component.Commit = latestReleaseCommit
+			component.Metadata.Tag = latestReleaseVersion
+			err = updateComponentsYaml(inputArgs.componentsConfigPath, components)
+			if err != nil {
+				exitWithError(errors.Wrap(err, "Failed to update components yaml"))
+			}
+
+			//run bump script
+			cmd := exec.Command("make", fmt.Sprintf("bump-%s", componentName))
+			if out, err := cmd.CombinedOutput(); err != nil {
+				exitWithError(errors.Wrapf(err, "Failed to run bump script. StdOut = %s", string(out)))
+			}
+
+			//create a new branch name
+			BranchName := strings.Replace(PRTitle, " ", "_", -1)
+			Logger.Printf("Opening new Branch %s", BranchName)
+
+			//push branch to PR
+
+		}
+	}
+}
+
+func getLatestReleaseInfo(componentUrl string) (string, string, error) {
+	return "", "", nil
+}
+
+func getCurrentReleaseTag(componentUrl string) (string, error) {
+	return "", nil
+}
+
+func isBumpNeeded(currentReleaseVersion, latestReleaseVersion, updatePolicy string) (bool, error) {
+	Logger.Printf("currentReleaseVersion: %s, latestReleaseVersion: %s, updatePolicy: %s\n", currentReleaseVersion, latestReleaseVersion, updatePolicy)
+	return false, nil
+}
+
+func initBumper(inputParams *inputParams) (*log.Logger) {
+	Logger = initLog()
+	// cold start message (visibly unique in log)
+	Logger.Printf("~~~~~~~~Init Bumper Script~~~~~~~~")
+	initFlags(inputParams)
+
+	return Logger
+}
+
+func initLog() *log.Logger {
+	var buf bytes.Buffer
+	logger := log.New(&buf, "INFO: ", log.LstdFlags)
+
+	file, err := os.OpenFile("bumper.logs", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0666)
+	if err != nil {
+		exitWithError(errors.Wrap(err, "failed to open log file bumper.logs"))
+	}
+	logger.SetOutput(file)
+
+	return logger
+}
+
+func initFlags(paramArgs *inputParams) {
+	flag.StringVar(&paramArgs.componentsConfigPath, "config-path", "", "Full path to components yaml")
+	flag.StringVar(&paramArgs.gitToken, "token", "", "git Token")
+	flag.Parse()
+	if flag.NFlag() != 2 {
+		exitWithError(fmt.Errorf("Wrong Number of input parameters %d, should be 2. Use --help for usage", flag.NFlag()))
+	}
+}
+
+func exitWithError(err error) {
+	fmt.Fprintf(os.Stderr, "Exiting with Error: %s\n", err)
```

Comment:
We don't need this is we are doing the logger

Reply:
moved logger to stdout, so this is redundant
Done